src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -171,7 +172,27 @@ const Footer = BaseFooter.extend({
         }
       });
     }
-  }
+  },
+
+  handleUpdateFooterLink() {
+    // only update link for loopback
+    if (!this.isFallbackApproach()) {
+      const link = getSignOutLink(this.options.settings, {
+        label: loc('loopback.polling.cancel.link.with.form.error', 'login')
+      })[0];
+      this.removeChildren();
```

Comment:
in future if this view contains some other link then this can cause issue.. See if you can remove only what you want to replace?